idc_index/index.py

```diff
@@ -918,15 +928,17 @@ def _parse_s5cmd_sync_output_and_generate_synced_manifest(
         with tempfile.NamedTemporaryFile(mode="w", delete=False) as synced_manifest:
             if dirTemplate is not None:
                 synced_df["s5cmd_cmd"] = (
-                    "sync " + synced_df["s3_url"] + " " + synced_df["path"]
+                    "sync " + synced_df["s3_url"] + " " + '"' + synced_df["path"] + '"'
                 )
                 list_of_directories = synced_df.path.to_list()
             else:
                 synced_df["s5cmd_cmd"] = (
-                    "sync " + synced_df["s3_url"] + " " + downloadDir
+                    "sync " + synced_df["s3_url"] + " " + '"' + downloadDir + '"'
                 )
                 list_of_directories = [downloadDir]
-            synced_df["s5cmd_cmd"].to_csv(synced_manifest, header=False, index=False)
+            synced_df["s5cmd_cmd"].to_csv(
+                synced_manifest, header=False, index=False, quoting=csv.QUOTE_NONE
```

Comment:
Can you explain the motivation and use of the `quoting` parameter?

Reply:
I had to dig deep for this one. When I add double quotes to the path and write to csv, csv contained another set of double quotes for each line.
something like "cp s3://idc-open-data/2629b726-1d5e-46bb-ad83-84e64c23309b/* "test"". 

 Apparently this is normal while writing to csv unless we disable with csv.QUOTE_NONE

Reply:
I read the commentary here https://www.perplexity.ai/search/explain-the-consequences-7kbYdW0xTUaERrsA3zj8AQ, and it comes with a lot of warnings. I need to think more about this...

Reply:
The thing is we *may* have strange characters in our table - DICOM `SeriesDescription` may as well have garbage..

Reply:
The current hierarchy does not include SeriesDescription.

I do not expect quotes in any of these
collection_id
PatientID
StudyInstanceUID
Modality
SeriesInstanceUID"

Reply:
bing suggested another simpler that does not need csv QUOTENone.

Reply:
You are right, I somehow thought about all of the columns.